src/debugger/debugviewcontroller.ts

```diff
@@ -167,15 +167,12 @@ class DebugViewController {
       case EntryCategory.TRACE:
         formattedData = this.serialiseTrace(entry)
         break
-
-      default:
-        throw new TypeError(`Unrecognised Entry Category: ${category}`)
     }
 
     const sessionTime = new Date(entry.sessionTime)
     const formatedSessionTime = `${formatDate(sessionTime)}.${zeroPadMs(sessionTime.getUTCMilliseconds())}`
 
-    return `${formatedSessionTime} - ${formattedData}`
+    return `${formatedSessionTime} - ${formattedData satisfies string}`
```

Comment:
Should this be `${formattedData} satisfies string` ?

Reply:
No, `[satisfies](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator) string` is part of the code